runtime/permissions.rs

```diff
@@ -27,6 +27,24 @@ use std::string::ToString;
 use std::sync::atomic::AtomicBool;
 #[cfg(test)]
 use std::sync::atomic::Ordering;
+#[cfg(not(windows))]
+use which::which;
+
+// `which::which` is bugged on Windows, we patch it here.
+// https://github.com/harryfei/which-rs/issues/54
+#[cfg(windows)]
+use which::which as which_;
+#[cfg(windows)]
+use which::Result as WhichResult;
```

Comment:
Can't these be moved inside the function to reduce the number of conditionals?

Reply:
One is in the signature